src/plugins/QuestScripts/QuestHooks.cpp

```diff
@@ -178,10 +178,10 @@ void DeeprunRatRoundup(Player* pPlayer, Object* pObject)
 	(TO_CREATURE(pObject))->SendChatMessage(CHAT_MSG_MONSTER_SAY, LANG_UNIVERSAL, "Into the box me pretties! Thas it. One by one ye go.");
 }
 
-void MaybellComplete(Player* pPlayer, Object* pObject)
+void TheEscape(Player* pPlayer, Object* pObject)
 {
-	(TO_CREATURE(pObject))->SendChatMessage(CHAT_MSG_MONSTER_YELL, LANG_UNIVERSAL, "Here goes nothing...");
-	(TO_CREATURE(pObject))->Emote(EMOTE_ONESHOT_CRY);
+	(TO_CREATURE(pObject))->SendChatMessage(CHAT_MSG_MONSTER_SAY, LANG_UNIVERSAL, "Here goes nothing...");
+	(TO_CREATURE(pObject))->SetOrientation(5.39307);
```

Comment:
Also you should use the method SetFacing() instead of SetOrientation(), otherwise the change will not be visible in the client. SetOrientation() just sets a field, SetFacing() also sends an SMSG_MONSTER_MOVE packet to the client.

Reply:
Altough even with SetFacing() I don't see the difference. Can you explain what is wrong with the orientation of that NPC @cressidagp ?

Reply:
OK, I can see now. The problem was that I was still in the dialog while she was trying to change orientation.
However I don't see why this is needed?
According to this video she drinks the invis. potion and that's it:
https://www.youtube.com/watch?v=ajs0F7nEu7U&t=89s

Reply:
> Altough even with SetFacing() I don't see the difference. Can you explain what is wrong with the orientation of that NPC @cressidagp ?

A saw a video when she chance orientación before drinks emote... But probably youre right. I Will remove that as soon i get home ;)